site/gatsby-site/playwright/e2e/integration/entityEdit.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { conditionalIntercept, waitForRequest, test, query } from '../../utils';
```

Comment:
Please use the `e2e-full` folder for new tests and do not rely on mocks, sorry that's been confusing.